src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyout.iOS.cs

```diff
@@ -61,18 +82,43 @@ private void BindToContent(string propertyName)
 		private void AttachAcceptCommand(IFrameworkElement rootControl)
 		{
 			var acceptButton = rootControl.FindName("AcceptButton") as Button;
+
 			if (acceptButton != null && acceptButton.Command == null)
-			{
-				acceptButton.Command = new DelegateCommand(Accept);
-				_hasAcceptButton = true;
-			}
+				acceptButton.Command = new DelegateCommand(Accept);		
+		}
+
+		private void AttachDismissCommand(IFrameworkElement rootControl)
+		{
+			var dismissButton = rootControl.FindName("DismissButton") as Button;
+
+			if (dismissButton != null && dismissButton.Command == null)
+				dismissButton.Command = new DelegateCommand(Dismiss);		
```

Comment:
brackets

Reply:
Find this convention overkill. Mostly half the code blocks are one liners, more so in well refactored code 

Might make it easier for code reviews/compare and merge but it simply add to much useless lines of code

i'll follow the convention but ..